base/docs/helpdb.jl

```diff
@@ -1466,11 +1466,25 @@ range
 doc"""
     eltype(type)
 
-Determine the type of the elements generated by iterating a collection of the given `type`. For associative collection types, this will be a `(key,value)` tuple type. The definition `eltype(x) = eltype(typeof(x))` is provided for convenience so that instances can be passed instead of types. However the form that accepts a type argument should be defined for new types.
+Determine the type of the elements generated by iterating a collection of the given `type`. For associative collection types, this will be a `Pair{keytype,valtype}`. The definition `eltype(x) = eltype(typeof(x))` is provided for convenience so that instances can be passed instead of types. However the form that accepts a type argument should be defined for new types.
```

Comment:
You should change the doc references of `keytype` and `valtype` so they aren't ambiguous with the function names. Probably some capitalization would do it.